src/Groups/group_characters.jl

```diff
@@ -3336,6 +3336,54 @@ function galois_orbit_sum(chi::GAPGroupClassFunction)
     return class_function(tbl, GAPWrap.ClassFunction(GapObj(tbl), GapObj(sums)))
 end
 
+
+@doc raw"""
+    galois_representative_and_multiplicity(chi::GAPGroupClassFunction; check::Bool = true)
+
+Return `phi, n, m` where `phi` is an absolutely irreducible constituent
+of the rational ordinary character `chi` and `n`, `m` are positive
+`ZZRingElem`s such that `phi` has `n` Galois conjugates
+and `chi` is `m` times the sum of these Galois conjugates.
```

Comment:
If `chi` is irreducible (as a rational character), is `m` the Schur index and `n` the degree of the character field of `phi`?

Reply:
> If `chi` is irreducible (as a rational character), is `m` the Schur index and `n` the degree of the character field of `phi`?

The character field of `phi` has degree `n`.

The multiplicity `m` is in general not the Schur index:
Take the irreducible character `chi` of degree 2 of the quaternion group of order 8. It is rational and irreducible, thus it is rationally irreducible, and has Schur index 2. The function returns `chi, 1, 1`.

Reply:
Makes sense, thanks for the explanation.

Reply:
I will add the above example to the docstring.